VMEncryption/main/DiskUtil.py

```diff
@@ -193,21 +194,37 @@ def secure_key_release_operation(self,protectorbase64,kekUrl,operation,attestati
         self.logger.log("secure_key_release_operation {0} end.".format(operation))
         return process_comm.stdout.strip()
     
-    def import_token(self,device_path,passphrase_file,public_settings):
+    def import_token_data(self,device_path,token_data,token_id):
```

Comment:
add documentation to the newly added, or touched methods to improve readability.

Reply:
I missed it. I will update.

Reply:
done

Reply:
Could you also add the sample token data for reference in the documentation?

Reply:
done